src/akushon/action/node/action_node.cpp

```diff
@@ -97,16 +97,42 @@ ActionNode::ActionNode(
       }
     );
   }
-}
 
-bool ActionNode::is_action_exist(int action_id) const
-{
-  return action_manager->get_action(action_id).get_name().empty();
+  {
+    using akushon_interfaces::srv::RunAction;
```

Comment:
Merge this service with the existing `RunAction` service. Modify the message so that the message can handle 2 types of `RunAction` methods, whether `RunAction` by name or `RunAction` by object of action.